### DISCUSSION TITLE
Shelly 2.5  WebUI - sliderbar not updated after command.

### DISCUSSION THREAD

**Discussion by mcmellow** [arendst/Tasmota#14520] [Q&A] [answered]
Shelly 2.5 with Version 2022.01.3  Date 2022-01-19T11:26:51

When I click on up/down button in the WebUI after the action the sliderbar is not updated. I have do a refresh (F5 key) before it is refreshed. The same happens when a open or closed is send from and other app.
Is there a setting I missed to have the sliderbar auto updated?

**Comment by sfromis** [comment, marked as answer]
Only that the Tasmota web UI is not really meant for "normal operation", only for admin/testing. That UI is deliberately rather rudimentary, and does not have full support for niceties like automatic refresh, and has rather slow updates anyway.

**Reply by mcmellow** [reply]
Thanks for the answer. As we use to say in the past: WAD = Works-As-Designed
